pkg/controller/consul_target_test.go

```
@@ -90,7 +88,7 @@ func (s *ConsulTargetSuite) TestCreate() {
 		s.NoError(err)
 		s.True(ok)
 
-		services, err := s.consulServer.Client.Agent().Services()
+		services, _ := s.consulServer.Client.Agent().Services()
```

I'm always :sus: of ignoring errors, but I guess in a test it's alright